### PR TITLE
chore: rename to etopay in sdk

### PR DESCRIPTION
**Pull request by kristifidani**
# Pull Request Template

## Motivation and Context

We decided to rename from Cawaena to ETOPay. Therefore, we need to do some renaming in the sdk. According to:
https://confluence.etogruppe.com/display/ZD0006/CIP-0024%3A+Rename+from+Cawaena+to+ETOPay

## Summary

- Renamed bindings crates and packages.
- Renamed sdk crate.
- Did also some renaming from the old `cryptpay` to `etopay`.
- Updated all the affected places.

This MR does not address all the changes required for renaming in sdk. But I decided to rename only the crates and packages here since it affects a lot of files.  Created some followup tickets.

Created 

- [ZD0006-621](https://jira.etogruppe.com/browse/ZD0006-621) to rename android bindings package and register the scope in Maven.
- [ZD0006-623](https://jira.etogruppe.com/browse/ZD0006-623) to register the scope in NPM.
- [ZD0006-622](https://jira.etogruppe.com/browse/ZD0006-622) to rename the public docs.

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [ ] Code follows project style and guidelines.
- [ ] No redundant or duplicate code.
- [ ] No added new dependencies without clear justification and approval.

### Documentation

- [ ] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] Added a CHANGELOG entry for major changes.

### Testing

- [ ] Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.
- [ ] Tests pass locally.

### Build & CI/CD

- [ ] Confirmed no CI/CD pipeline issues.
- [ ] Updated build scripts where necessary.

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -190,7 +190,7 @@ jobs:
       - name: Upload Artifact
         uses: actions/upload-artifact@v4
         with:
-          name: cawaena-sdk-wasm
+          name: etopay-sdk-wasm
```
Comment (antbern) [inline .github/workflows/ci.yml:193]: I cannot put a comment on the affected line, but further up in this job definition we have `--out-name cryptpay-sdk-node` which should probably be changed 💯 (Or in this case we could probably remove the `--out-name` parameter completely)

There is also `--scope cawaena` which perhaps we should change?

**Review by antbern — COMMENTED** [review]
Looking good, just one small comment!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kristifidani) [reply .github/workflows/ci.yml:193]: Nice, removed the `--out-name` param and renamed the `--scope etopay` 👍  

Updated also the swift binary

**Review by antbern — APPROVED** [review]

**Review by antbern — APPROVED** [review]

**Comment by kristifidani** [comment]
@ETOSPHERES-Labs/lobster  

I will stop making changes here now so would appreciate a review and merge this so we can continue with the rest of the changes. Created some followup-tickets and added them in the sprint:

- [ZD0006-621](https://jira.etogruppe.com/browse/ZD0006-621) to rename android package and register the scope in Maven.
- [ZD0006-623](https://jira.etogruppe.com/browse/ZD0006-623) to register the scope in NPM.
- [ZD0006-622](https://jira.etogruppe.com/browse/ZD0006-622) to rename the public docs.

**Review by antbern — APPROVED** [review]

**Review by hauju — APPROVED** [review]
